core/storage/feedback/gae_models.py

```diff
@@ -81,6 +81,18 @@ def get_deletion_policy():
         """General feedback thread needs to be pseudonymized for the user."""
         return base_models.DELETION_POLICY.LOCALLY_PSEUDONYMIZE
 
+    @classmethod
+    def has_reference_to_user_id(cls, user_id):
+        """Check whether GeneralFeedbackThreadModel exists for user.
+
+        Args:
+            user_id: str. The ID of the user whose data should be checked.
+
+        Returns:
+            bool. Whether any models refer to the given user ID.
+        """
+        return cls.query(cls.original_author_id == user_id).get() is not None
```

Comment:
I have a question over here. On wipeout, do all feedback threads/suggestions/messages created by a user get deleted or do they become anonymous? On deleting a feedback thread, linked messages and stats models created would also need to be deleted then right? 

(This comment does not directly relate to this PR, but just got the question while I was reviewing)

Reply:
They get anonymized, all the related models will also be handled.